test/APINotes/Inputs/custom-frameworks/APINotesFrameworkTest.framework/Headers/APINotesFrameworkTest.apinotes

```diff
@@ -259,7 +259,7 @@ SwiftVersions:
         SwiftName: multiVersionedGlobal45Notes_5
       - Name: multiVersionedGlobal45Both
         SwiftName: multiVersionedGlobal45Both_5
-  - Version: 4 # Versions are deliberately ordered as "3, 5, 4" to catch bugs.
+  - Version: 4 # Versions are deliberately ordered as "3, 5, 4.2, 4" to catch bugs.
```

Comment:
Can you add a separate test that shows that "Version: 4.2" with no other versions specified also affects `-swift-version 4` and `-swift-version 3`?, and that "Version: 4" with no other versions does *not* affect `-swift-version 4.2`? Or maybe that's in here already; the original 3-4-5 tests did do things like that.

Reply:
@jrose-apple I added this in the latest commit.